htdocs/lib2/logic/cachelist.class.php

```diff
@@ -20,7 +20,7 @@ class cachelist
     public $nCachelistId = 0;
     public $reCachelist;
 
-    public function __construct($nNewCachelistId = ID_NEW, $nUserId = 0)
+    public function __construct($nNewCacheListId = ID_NEW, $nUserId = 0)
```

Comment:
"Cachelist" is a fixed term here ("Cacheliste"), should stay this way to avoid confusion with some aggegate data type. The name of the class then should be "Cachelist" later.